tests/ui/test_ui_tools.py

```diff
@@ -463,10 +463,10 @@ def test_init(self, mocker, stream_view):
         ('here', ['test here'], []),
         ('test here', ['test here'], []),
         # With 'foo' pinned.
-        ('f', ['foo', '', 'fan', 'FOO', 'FOOBAR'], [['foo'], ]),
-        ('FOO', ['foo', '', 'FOO', 'FOOBAR'], [['foo'], ]),
+        ('f', ['foo', '', 'fan', 'FOO', 'FOOBAR'], ['foo', ]),
+        ('FOO', ['foo', '', 'FOO', 'FOOBAR'], ['foo', ]),
```

Comment:
These are changing from the full pinned stream data (abbreviated to just the name - first entry), to just the names (to generate the data in the test)?